libs/blocks/aside/aside.css

```diff
@@ -859,7 +859,6 @@
   }
 
   .aside.notification.small .foreground.container .text .icon-area {
-    width: 40px;
```

Comment:
Should there be a max-width, though?

Reply:
@overmyheadandbody there actually is one that cascades to this variant, rule is on line 199 of aside.css